treeherder/perf/fixtures/performance_bug_templates.yaml

```diff
@@ -7,7 +7,11 @@
     default_product: Firefox
     cc_list: jmaher@mozilla.com, rwood@mozilla.com, ionut.goldan@softvision.ro
     text: |
-      Talos has detected a Firefox performance regression from push {{ revision }}. As author of one of the patches included in that push, we need your help to address this regression.
+      Talos has detected a Firefox performance regression from push
```

Comment:
There should be a colon (:) at the end of this line (and subsequently).

e.g.

`Talos has detected a Firefox performance regression from push:`